juju/juju
# migration for subnet.SpaceID

Requires https://github.com/juju/description/pull/57

## Description of change

Add migration steps for subnet.SpaceID.  If migrating forward, the spaceID will be the default and can be corrected with juju reload-spaces.

## QA steps

1. juju bootstrap maas with multiple spaces in 2.6.5.  
2. juju add-model move-me
2. juju bootstrap maas with multiple spaces with this change.
3. juju migrate move-me <2nd controller>
2. juju switch <2nd controller>
3. notice all subnets in default space.
2. juju add-model move-me-again
3. juju bootstrap maas with multiple spaces with this change.
3. juju migrate move-me <3rd controller>
2. juju switch <3rd controller>
3. notice all subnets in same space as on controller 2.
